chia/_tests/core/util/test_streamable.py

```diff
@@ -420,6 +447,12 @@ def validate_item_type(type_in: type[Any], item: object) -> bool:
             list_type = get_args(type_in)[0]
             assert type(item) is list
             return all(validate_item_type(list_type, list_item) for list_item in item)
+        if is_type_Dict(type_in):
+            [key_type, value_type] = get_args(type_in)
+            assert type(item) is dict
```

Comment:
it's not obvious why the `item` has to be of type `dict`. Is that even possible? if you can only stream dicts whose value is another dict, which (presumably) in turn has a value of dict, and so on indefinitely. Am I missing something?
I would have thought you would just check that both `key` and `value` are streamable